staging/src/k8s.io/kubectl/pkg/cmd/get/sorter.go

```diff
@@ -264,7 +279,17 @@ func isLess(i, j reflect.Value) (bool, error) {
 			}
 		case string:
 			if jtype, ok := j.Interface().(string); ok {
-				return sortorder.NaturalLess(itype, jtype), nil
+				// check if it's a Quantity
```

Comment:
Is it actually possible to hit both this case and the one at L185? I'm new to this code, but since the target is always structured the same, I'd expect to always hit the same case (well, one consistent case for a unstructured.Unstructured resource input and a second consistent case for a proper type containing resource.Quantity, as discussed in the other thread). Experimentation suggests to me that this is the one hit by the Unstructured case.

Reply:
I was not sure if it will hit the case `reflect.String` so I added that in just in case, will remove that 😄 . I am not sure what is annotation, can you clarify that?

Reply:
You are right, it's not needed 👍🏼